### DISCUSSION TITLE
Windows 10 Multiple USD Environment conflicts

### DISCUSSION THREAD

**Discussion by moo-bius** [Autodesk/maya-usd#3360] [Q&A] [answered]
Hello,

I have the standard version of USD installed which is causing conflicts with MayaUSD preventing the plugin from working due to the environment conflicts. I don't want to replace my standard USD with MayaUSD in my system variables.

 Is there anything to fix this? Maya.env and userSetup.py havn't helped.

I'm not well versed in messing with environment variables.

Thanks,

**Comment by seando-adsk** [comment, marked as answer]
Are you working on the Windows platform? The important thing on Windows is to **not** have any other USD in your system environment variables PATH and PYTHONPATH. You can edit the environment variables from the "System Properties". Make sure that both the PATH and PYTHONPATH for User and System do not have any USD in them.

You should also never need to add any MayaUsd paths (including the USD contained in it) to any system environment variable. When you install MayaUsd there is a .mod file installed that Maya will read to setup MayaUsd when starting Maya.

Sean
